inst/qview/qview.R

```diff
@@ -0,0 +1,92 @@
+library(shiny)
+library(tidyverse)
+library(DT)
+
+
+#' queueViewOutput
+#' @export
+queueViewOutput <- function(id) {
+  ns <- NS(id)
+  tableOutput(ns("queue"))
+}
+
+#' #' queueButtonInput
+#' #' @export
+#' queueButtonInput <- function(id, label, job_id) {
+#'   ns <- NS(id)
+#'   actionButton(
+#'     inputId = ns(id),
+#'     label = label,
+#'     icon = icon("stop-circle"),
+#'     job_id = job_id
+#'   )
+#' }
+
+#' queueView
+#' @export
+queueView <- function(input, output, session, queue_obj = rrsq::RSimpleQueue$new()) {
+  output$queue <- renderTable(
+    {
+      invalidateLater(1000, session)
+
+      queueViewFrame <- rrsq::jobs_to_df(queue_obj$get_jobs())
+
+      return(
+        queueViewFrame
+      )
+    }
+  )
+}
+
+
+## Testing ####
+
+action_button_creator <- function(index) {
+  return(
+    as.character(
+      actionButton(
+        as.character(index),
```

Comment:
I realized I forgot to address this -- I agree, but since we're making this into a module, I think the module namespacing makes this less of a concern. I'll look into fixing it.